pocs/camera/simulator_sdk/ccd.py

```diff
@@ -34,6 +34,7 @@ def __init__(self,
         self._is_cooled_camera = True
         self._cooling_enabled = False
         self._temperature = 25 * u.Celsius
+        self._target_temperature = target_temperature
```

Comment:
I tried to do a `kwargs.update({'target_temperature': target_temperature})` before the `super()` call but it's not working for some reason. I think that would be preferred so will try to look at how the inheritance is working later.